docs/preview/03-Features/correlation.md

```diff
@@ -91,18 +70,19 @@ builder.Services.AddHttpCorrelation();
 WebApplication app = builder.Build();
 app.UseHttpCorrelation();
 app.UseRouting();
+app.UseRequestTracking();
```

Comment:
Should we add a little additional paragraph on this ?
Saying that calling this method makes sure that middleware is being configured that makes sure that incoming requests are automatically tracked ?

Reply:
We could, but this is already explained in a dedicated section. Maybe we should point to the logging page instead?

Reply:
Which section ?
We could, but I think one or two sentences that just explain that this line makes sure that incoming requests (and outgoing dependencies?) are automatically tracked, might be enough.

Reply:
>  (and outgoing dependencies?)

The `UseHttpCorrelation()` makes sure of that, the `UseRequestTracking()` only tracks the request.
Added line here: https://github.com/arcus-azure/arcus.webapi/pull/392/commits/ad916a1d9c5b4bbb949bb42671a4bacd96f5dc8b